### PR TITLE
Add notes to KDE PIM

### PR DESCRIPTION
**Pull request by lbeltrame**
As requested in http://bugzilla.suse.com/show_bug.cgi?id=1006586, this is a first integration of ntoes regarding KDE PIM in Leap 42.2. I'm not sure it's the right section, but skimming through the document, I could not find anything that would fit.

P.S.: First time doing DocBook, bear that in mind.

### CODE REVIEW

File: xml/release-notes.xml [hunk 1]
```diff
@@ -618,13 +618,41 @@
...
   <title>General</title>
 
   <para>
     This section lists general issues with &opensuseleap;
     &version; that do not match any other category.
-  </para> -->
+  </para>
+
+  <section xml:id="sec.general.kdepim">
+      <title>KDE Software for personal information management (KDE PIM)</title>
```
Comment (Vogtinator) [inline xml/release-notes.xml]: Titles are usually written capitalized, so this should be ```KDE Software for Personal Information Management (KDE PIM)```

File: xml/release-notes.xml [hunk 2]
```diff
@@ -618,13 +618,41 @@
...
+      <para>
+          &opensuseleap; &version; ships two versions of the KDE PIM
+          (kontact, kmail, etc.) suite: the first is the old legacy 4.x
```
Comment (Vogtinator) [inline xml/release-notes.xml]: ```old legacy``` sounds like a duplicate, what about just ```legacy```?

File: xml/release-notes.xml [hunk 3]
```diff
@@ -618,13 +618,41 @@
...
+          version and the second is the one based on KDE Frameworks 5.
+          The former is no longer supported by upstream KDE, but it was
+          kept not to distrupt user workflows.
+      </para>
+      <para>
+          The two versions are incompatible with each other, meaning that
```
Comment (Vogtinator) [inline xml/release-notes.xml]: I'd rephrase this as ```The two versions of KDE PIM are not co-installable.```

File: xml/release-notes.xml [hunk 4]
```diff
@@ -618,13 +618,41 @@
...
+          only one can be installed at a time. Some software, such as
+          <package>knode</package> requires specifically the legacy 4.x
+          version and will be uninstalled if any package from KDE PIM 5.x
+          (for example <package>kmail5</package> will be installed.
+      </para>
+      <para>
+          It is important to note that KDE PIM 4.x is effectively marked as
```
Comment (Vogtinator) [inline xml/release-notes.xml]: Why ```effectively```?

File: xml/release-notes.xml [hunk 5]
```diff
@@ -618,13 +618,41 @@
...
+          deprecated, and will be removed in the next version of &opensuseleap;
+      </para>
+      <para>
+          Users are encouraged to switch to the newer 5.x version, keeping
+          in mind that not all settings are migrated at this time (see
+          <link xlink:href="http://bugzilla.opensuse.com/show_bug.cgi?id=1001872"/>)
```
Comment (Vogtinator) [inline xml/release-notes.xml]: This link is broken, there is no ```opensuse.com```.

**Comment by sknorr** [comment]
Thanks! Merged.
